test/gtest/Makefile.am

```diff
@@ -43,7 +43,7 @@ gtest_CPPFLAGS = \
 	$(GTEST_CPPFLAGS)
 
 gtest_LDFLAGS  = $(GTEST_LDFLAGS) -no-install
-gtest_CXXFLAGS = $(GTEST_CXXFLAGS) -g -O3
+gtest_CXXFLAGS = $(GTEST_CXXFLAGS) -g -O3 -fno-tree-vectorize
```

Comment:
i got segfault when adding callbacks for user-defined type. seems like gcc assumes the pack destination buffer is aligned and generates SSE instructions which assume memory is aligned (e.g. movdqa)